### PR TITLE
[package] Add better debugging for torch.package

### PR DESCRIPTION
**Pull request by PaliC**
Summary:
Makes torch.package debugging more transparent by
1. Pointing out not implictily externed modules in the standard library.
2. Creating a debug mode for users to find the source of broken modules.

Test Plan: Run package tests

Differential Revision: D42728753

### CODE REVIEW

File: torch/package/package_exporter.py [hunk 1]
```diff
@@ -154,7 +154,25 @@ def __init__(self, dependency_graph: DiGraph):
...
+                    f"level access and therefore can pose a security risk. We heavily "
+                    f"recommend removing `{module_name}` from your packaged code. However, if that "
+                    f"is not possible, add it to the extern list by calling "
+                    f"PackageExporter.extern(\"`{module_name}`\")\n"
+                    )
+                    )
+                if debug:
+                    module_path = dependency_graph.first_path(module_name)
+                    message.write(f"      A path to {module_name}: {' -> '.join(module_path)}")
+        if not debug:
```
Comment (kurman) [inline torch/package/package_exporter.py:171]: shouldn't this be in the for loop?

Comment (PaliC) [reply torch/package/package_exporter.py:171]: This error message specifically isn't related to an individual module and is just advertising the debug functionality. It wouldn't make sense to stick it in the loop.

**Review by kurman — APPROVED** [review]

**Pull request closed without merging**